bip-0352.mediawiki

```diff
@@ -303,6 +303,70 @@ If using a seed phrase only style backup, the user can recover the wallet's unsp
 
 Silent payments introduces a new address format and protocol for sending and as such is not compatible with older wallet software or wallets which have not implemented the silent payments protocol.
 
+== Test Vectors ==
+
+A [[bip-0340/test-vectors.csv|collection of test vectors in JSON format]] are provided, along with a [[bip-0340/reference.py|python reference implementation]]. Each test vector consists of a sending test case and corresponding receiving test case. This is to allow sending and receiving to be implemented separately. Test cases use the following schema:
```

Comment:
I think links are wrong here :)

Reply:
Thanks! That is indeed the wrong link, will update!